app/src/main/java/com/technopark/hw_1/FirstFragment.java

```diff
@@ -0,0 +1,96 @@
+package com.technopark.hw_1;
+
+import android.content.res.Configuration;
+import android.os.Bundle;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.Button;
+
+import androidx.annotation.NonNull;
+import androidx.annotation.Nullable;
+import androidx.fragment.app.Fragment;
+import androidx.recyclerview.widget.GridLayoutManager;
+import androidx.recyclerview.widget.RecyclerView;
+
+import org.jetbrains.annotations.NotNull;
+
+import static com.technopark.hw_1.Consts.*;
+
+interface OnNumberClickListener {
+    void onNumberClick(NumberModel numberModel);
+}
+
+
+public class FirstFragment extends Fragment implements OnNumberClickListener {
```

Comment:
I suggest to use clearer name, e.g. `NumberListFragment` or `ListFragment`. Now `FirstFragment` does not make sense at all